libs/openFrameworks/graphics/ofBitmapFont.cpp

```diff
@@ -418,11 +418,11 @@ ofMesh ofBitmapFont::getMesh(const string & text, int x, int y, ofDrawBitmapMode
 		if(text[c] == '\n'){
 
 			sy += lineHeight*newLineDirection;
-			if(mode == OF_BITMAPMODE_SIMPLE) {
+//			if(mode == OF_BITMAPMODE_SIMPLE) {
```

Comment:
I didn't remove this because I really don't understand why it it is there in first place although the most logical thing is that it shouldn't be there. If there's no problem removing it I will.